container-scan/action.yaml

```diff
@@ -68,11 +123,11 @@ runs:
     - name: Check for previous report comment
       id: fc
       if: github.event.pull_request.number != ''
```

Comment:
I was executing this action on a repo manually (workflow_dispatch) and since it wasn't being triggered in a pull request context or PR number was missing it failed. 

Maybe we need to add a condition here 
```
if: github.event_name == 'pull_request'